docusign/docusign.lua

```diff
@@ -0,0 +1,79 @@
+--[[
+
+# [DocuSign API DOCUMENTATION](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](https://admindemo.docusign.com).
+
+## Input parameters
+
+* `username` - application user email used to authorize the call.
+* `password` - user password used to authorize the call
+* `integratorKey` - integration and authenticate with the DocuSign platform
+
+* api_endpoint - API endpoint URI, e.g. `user`.
+* method - optional request method, defauls to `GET`, could be `GET`, `POST`, `PUT`, `DELETE`.
+* post_data - optional JSON string used to post data, e.g. `{"blog":{"title":"Test Title"}}` structure used to create a Blog item.
+
+## Output parameters
+
+All connector output parameters could be used in PMIO Service Task output parameters as a value placeholders in curly braces.
+
+Example connector output parameters:
+
+```
+name = {response.name}
+accountId = {response.accountId}
+```
+
+* response - JSON decoded structure received from API response.
+* code - HTTP response code.
+* headers - headers structure received from API response.
+
+]]--
+
+local https = require("ssl.https")
+local ltn12 = require"ltn12"
+local cjson = require("cjson")
+
+local mime = require("mime")
+
+local inputVar = cjson.decode(io.stdin:read("*a"))
+
+local respbody = {}
+local reqbody = inputVar.post_data and cjson.encode(inputVar.post_data) or ''
+
+local url = 'https://demo.docusign.net/restapi/v2' .. '/' .. inputVar.api_endpoint
```

Comment:
This connector is useless for real usage, please refer to 
https://docs.docusign.com/esign/guide/usage/quickstart.html#authenticate-and-get-your-base-url
You hardcoded demo domain, disallowing users go on production.

add `baseURL` input parameter